st2common/st2common/models/db/execution.py

```diff
@@ -182,6 +182,7 @@ class ActionExecutionOutputDB(stormbase.StormFoundationDB):
     delay = me.IntField()
 
     data = me.StringField()
+    next_tasks = me.StringField()
```

Comment:
Please also update class docstring comment which clarifies what this attribute is used for / what value is contains.

Reply:
This model should be "output data generic" so I'm not sure if adding this attribute to it directly is the best approach.

Instead, one option would be to insert a new ``ActionExecutionOutputDB`` document for ``next_task`` purposes where you could set ``output_type`` to ``workflow.next_tasks`` (or similar) and ``data`` field would contain the actual value.

This way we preserve the generic nature of that model.